libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -21,41 +21,43 @@ const singleRow: Accounts_party_accounts = {
 };
 const singleRowData = [singleRow];
 
-it('should render successfully', async () => {
-  await act(async () => {
-    const { baseElement } = render(<AccountsTable data={[]} />);
-    expect(baseElement).toBeTruthy();
+describe('AccountsTable', () => {
```

Comment:
jus added `describe(..` ad updated `act(async`  after the testing library upgrade.